filebeat/input/v2/input-cursor/store.go

```diff
@@ -137,7 +157,9 @@ func (s *store) Release() {
 }
 
 func (s *store) close() {
-	panic("TODO: implement me")
+	if err := s.persistentStore.Close(); err != nil {
+		s.log.Errorf("Closing registry store did report an error: %+v", err)
```

Comment:
Is error displayed with `%+v` on purpose?

Reply:
Yes. Logging errors is a 'pain' as there is no common way of formatting. Some error types only report the cause when '+' or '#' is given. Unfortunately some error package will include a stacktrace, others not